salt/modules/config.py

```diff
@@ -561,4 +561,4 @@ def items():
 
         salt '*' config.items
     """
-    return __opts__
+    return __opts__.value()
```

Comment:
I don't think it's always a `NamedLoaderContext`.

```suggestion
    try:
        return __opts__.value()
    except AttributeError:
        return __opts__
```

Reply:
Do you know in what cases it could be not `NamedLoaderContext`? As far as I know `__opts__` is populated with `LazyLoader` and to be fair I'm not sure if it can push something different there.

Reply:
Tbh now I'm not sure anymore if what I said is correct. I looked up what I remembered when writing the above, but it might have been Salt-SSH-specific only. Sorry for the noise! :)

In general, I think one of the cases where this might happen in a regular module is if the dunder is initialized to a dict during module import in the top-level module code, but that's not the case here (on the contrary).